index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Overnight Oats</title>
+  </head>
+
+  <body>
+    <div class="card">
+      <div class="image-wrapper">
+        <img
+          class="image"
+          src="https://cdn.glitch.com/9ad19879-94c0-4d8f-a4cd-f81b103e619c%2FIMG_0795.jpg?v=1602715083489"
+          alt="Overnight Oats in Jars with rasperies on top"
+        />
+      </div>
+      <div class="content">
+        <h2>Overnight Oats</h2>
+        <section id="info" class="information">
+          <ul>
+            <li>
+              <img src="https://cdn.glitch.com/9ad19879-94c0-4d8f-a4cd-f81b103e619c%2FAsset%203.png?v=1604348855512" alt="serving size icon">
+              2
+            </li>
+            <li>
+              <img src="https://cdn.glitch.com/9ad19879-94c0-4d8f-a4cd-f81b103e619c%2FAsset%204.png?v=1604348837107" alt="prep time icon">
+              15min
+            </li>
+            <li>
+              <img src="https://cdn.glitch.com/9ad19879-94c0-4d8f-a4cd-f81b103e619c%2FAsset%206.png?v=1604349326136" alt="caloric info icon"> 
+              105
+            </li>
+          </ul>
+        </section>
+        <nav>
+          <ul>
+            <li>
+              <a href="#directions" class="button">
+                <img
+                  src="https://cdn.glitch.com/9ad19879-94c0-4d8f-a4cd-f81b103e619c%2FAsset%205.png?v=1604348913969"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="#ingredients" class="button">
+                <img
+                  src="https://cdn.glitch.com/9ad19879-94c0-4d8f-a4cd-f81b103e619c%2FAsset%201.png?v=1603844512332"
+              /></a>
+            </li>
+          </ul>
+        </nav>
+        <section id="ingredients" class="toggle-content">
+          <h3>Ingredients</h3>
+          <ul>
+            <li>2 servings of fresh fruit</li>
+            <li>200ml of milk</li>
+            <li>80g of fine oats</li>
+          </ul>
+        </section>
+        <section id="directions" class="toggle-content">
+          <h3>Directions</h3>
+          <ol>
+            <li>
+              Put oats, some toppings, some fruits and sweetner in a bowl. Fill
+              up with milk and stir then store in fridge overnight.
+            </li>
+            <li>
+              The next morning serve with the rest of the fruits, toppings, and
+              spices.
+            </li>
+            <li>Enjoy!</li>
+          </ol>
+        </section>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Your overall HTML is very tidy and uses container elements (like `section` and `div`) appropriately. Great work!